centml/cli/cluster.py

```diff
@@ -71,43 +108,146 @@ def get(id):
     )
 
     click.echo("Additional deployment configurations:")
-    click.echo(
-        tabulate(
-            [
-                ("Is private?", deployment.secrets is not None),
-                ("Hardware", id_to_hw_map[deployment.hardware_instance_id]),
-                ("Port", deployment.port),
-                ("Healthcheck", deployment.healthcheck or "/"),
-                ("Replicas", {"min": deployment.min_replicas, "max": deployment.max_replicas}),
-                ("Environment variables", deployment.env_vars or "None"),
-            ],
-            tablefmt="rounded_outline",
-            disable_numparse=True,
+    if type == platform_api_client.DeploymentType.INFERENCE:
+        click.echo(
+            tabulate(
+                [
+                    ("Port", deployment.port),
+                    ("Healthcheck", deployment.healthcheck or "/"),
+                    ("Replicas", {"min": deployment.min_replicas, "max": deployment.max_replicas}),
+                    ("Environment variables", deployment.env_vars or "None"),
+                    ("Max concurrency", deployment.timeout or "None"),
+                ],
+                tablefmt="rounded_outline",
+                disable_numparse=True,
+            )
+        )
+    elif type == platform_api_client.DeploymentType.COMPUTE:
+        click.echo(
+            tabulate(
+                [
+                    ("Port", deployment.port),
+                    ("Username", deployment.username or "None"),
+                    ("SSH key", deployment.ssh_key or "None"),
```

Comment:
lets not show the whole ssh key. similar to the UI, we should trim it at some length

Reply:
done , similar to UI i have added the same logic for trim and fallback message if that's not available